deprecated/notebooks/.ipynb_checkpoints/HT_OWID_manipulation-checkpoint.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
add `.ipynb_checkpoints` to gitignore and remove these temporary files.

Reply:
`.ipynb_checkpoints` was just added to `.gitignore`. Do you mean these two `untitled.ipynb` within [`deprecated/notebooks`](https://github.com/hongtaoh/covid19-data/tree/master/deprecated/notebooks)? I am not quite sure which files to remove.

Reply:
files within the `.ipynb_checkpoints` folder, not within notebooks folder.

Reply:
Got it! I just have removed the `ipynb_checkpoints` folder within `deprecated/notebooks`.  Sorry for all mistakes made.